app/frontend/src/components/forms/manage/ManageForm.vue

```diff
@@ -128,21 +166,24 @@ import { FormPermissions, NotificationTypes } from '@/utils/constants';
 import ApiKey from '@/components/forms/manage/ApiKey.vue';
 import FormSettings from '@/components/designer/FormSettings.vue';
 import ManageVersions from '@/components/forms/manage/ManageVersions.vue';
+import Subscription from '@/components/forms/manage/Subscription.vue';
 
 export default {
   name: 'ManageForm',
-  components: { ApiKey, FormSettings, ManageVersions },
+  components: { ApiKey, FormSettings, ManageVersions, Subscription },
   data() {
     return {
       apiKeyPanel: 1,
       formSettingsDisabled: true,
       settingsFormValid: false,
       settingsPanel: 1,
       versionsPanel: 0,
+      subscriptionsPanel: 0,
+      subscription: false,
     };
   },
   computed: {
-    ...mapGetters('form', ['apiKey', 'drafts', 'form', 'permissions', 'isRTL']),
+    ...mapGetters('form', ['apiKey', 'drafts', 'form', 'permissions', 'isRTL', 'subscriptionData']),
```

Comment:
Looks like a prettier complaint - some of us have prettier configured to format on save, but running the unit tests will also run prettier and will format this line.